src/ProvidedTypes.fs

```diff
@@ -8043,10 +8081,10 @@ namespace ProviderImplementation.ProvidedTypes
         override __.ReflectionOnly = true
 
         override x.GetManifestResourceStream(resourceName:string) =
-            //let r = getReader().ILModuleDef.Resources.Entries |> Seq.find (fun r -> r.Name = resourceName)
-            //match r.Location with
-            //| ILResourceLocation.Local f -> new MemoryStream(f()) :> Stream
-            //| _ -> 
+            let r = getReader().ILModuleDef.Resources.Entries |> Seq.find (fun r -> r.Name = resourceName)

```

Comment:
@dsyme I'm not sure about the cost of `getReader .. Entries`, but wondering if it is amortized.
What is the implication in case the `GetManifestResourceStream` method is repeatedly called with same or different values?

Reply:
> What is the implication in case the GetManifestResourceStream method is repeatedly called with same or different values?

It will repeatedy re-read and GC the old readers.